### DISCUSSION TITLE
Errors in the Template Editor are making the template not work

### DISCUSSION THREAD

**Discussion by hsuantishih** [windingwind/zotero-better-notes#1104] [Q&A]
Firstly, thank you for the best work on this plugin. I'm new to Zotero and met some problems with the template. I can't insert any of them into my notes. I hope anyone can help me out. I appreciate it.
![image](https://github.com/user-attachments/assets/a8fefd3c-1bf0-4467-8389-c584f9778f16)

**Comment by windingwind** [comment]
If you think it’s a plugin bug, please report a new issue. Close here.

**Reply by hsuantishih** [reply]
Thank you for your reply. I tried reinstalling it, and it worked.
